srcpkgs/libstemmer/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'libstemmer'
+pkgname=libstemmer
+version=2.1.0
+revision=1
+wrksrc="snowball-${version}"
+build_style=gnu-makefile
+build_helper=qemu
+hostmakedepends="perl"
+short_desc="Small string processing language for creating stemming algorithms"
+maintainer="Michal Vasilek <michal@vasilek.cz>"
+license="BSD-3-Clause"
+homepage="https://snowballstem.org/"
+distfiles="https://github.com/snowballstem/snowball/archive/refs/tags/v${version}.tar.gz"
+checksum=24ae0b28d6531190c2cd960d515ec9374d3a3d9420c4a0accdf89b7f02fc9caa
+patch_args="-Np1"
+make_check=no # tests fail
+
+post_patch() {
+	if [ -n "$CROSS_BUILD" ]; then
+		vsed -i GNUmakefile -e 's|\t\./snowball|\tqemu-${XBPS_TARGET_QEMU_MACHINE}-static ./snowball|'
+	fi
+}
+
+do_install() {
+	vinstall include/libstemmer.h 644 usr/include
+	vinstall libstemmer.so.0.0.0 644 usr/lib
+	ln -s libstemmer.so.0.0.0 ${DESTDIR}/usr/lib/libstemmer.so
+	ln -s libstemmer.so ${DESTDIR}/usr/lib/libstemmer.so.0
```

Comment:
TBH I am not a fan of this. Can the library be statically linked? How are other distros handling this situation?

Reply:
@paper42, please document that other distros are doing this, so the library is being treated as shared by downstreams. I'd appreciate links in the commit to the packaging done by other distros too.

IMO this should have a big `# XXX` comment.

Why are you against statically linking it? Kinda sucks for size, but we won't be caught by any unexpected upstream breakage, at least.

Reply:
> @paper42, please document that other distros are doing this, so the library is being treated as shared by downstreams. I'd appreciate links in the commit to the packaging done by other distros too.
> 
> IMO this should have a big `# XXX` comment.

done

> Why are you against statically linking it? Kinda sucks for size, but we won't be caught by any unexpected upstream breakage, at least.

If you think it would be a better idea, I can do it, but no other distribution I looked at does this and it would require extra care in each package that depends on libstemmer (I expect there will be more dependants than just geary and calibre in the future).